app/src/main/java/org/p2p/wallet/solend/ui/deposit/SolendDepositFragment.kt

```diff
@@ -0,0 +1,125 @@
+package org.p2p.wallet.solend.ui.deposit
+
+import android.os.Bundle
+import android.view.View
+import org.koin.android.ext.android.inject
+import org.koin.core.parameter.parametersOf
+import org.p2p.uikit.glide.GlideManager
+import org.p2p.uikit.utils.toPx
+import org.p2p.wallet.R
+import org.p2p.wallet.common.mvp.BaseMvpFragment
+import org.p2p.wallet.databinding.FragmentSolendDepositBinding
+import org.p2p.wallet.solend.model.SolendDepositToken
+import org.p2p.wallet.solend.ui.bottomsheet.SelectDepositTokenBottomSheet
+import org.p2p.wallet.utils.args
+import org.p2p.wallet.utils.formatToken
+import org.p2p.wallet.utils.popBackStack
+import org.p2p.wallet.utils.scaleShort
+import org.p2p.wallet.utils.viewbinding.viewBinding
+import org.p2p.wallet.utils.withArgs
+import java.math.BigDecimal
+
+private const val ARG_DEPOSIT_TOKEN = "ARG_DEPOSIT_TOKEN"
+
+private const val KEY_REQUEST_TOKEN = "KEY_REQUEST_TOKEN"
+private const val KEY_RESULT_TOKEN = "KEY_RESULT_TOKEN"
+
+class SolendDepositFragment :
```

Comment:
SolendUserDepositDetailedFragment?
we already have SolendUserDeposits - the list
would be nice to have same naming for the detailed view

Reply:
it is not details
it is a fragment where you can deposit token =)